### DISCUSSION TITLE
Ldot

### DISCUSSION THREAD

**Discussion by colinmford** [koeberlin/Latin-Character-Sets#2] [General]
Thanks for making this resource publicly available!

In [README.md](https://github.com/koeberlin/Latin-Character-Sets#l_periodcenteredlatncatl_periodcenteredlatncat), you mention you prefer to not use encoded characters for the Catalan Ldot:

> /L_periodcentered.latnCAT/l\⁠_periodcentered.latnCAT
> 
> I recommend to include these glyphs instead of the encoded characters /Ldot/ldot (0x013F, 0x0140).

Can you explain your reasoning behind this choice a little more? Is including the encoded characters actually harmful?

**Comment by koeberlin** [comment]
Many thanks for your kind feedback, Colin!

I generally try to avoid encoded glyphs in substitution features to preserve text semantics (also f_f_i instead of uniFB03), but I may be too cautious. Including the encoded character shouldn’t be harmful in any case!

I hope for more feedback by Catalan speakers to sharpen this advice!

**Comment by weiweihuanghuang** [comment]
Can you please clarify what you mean by preserve 'text semantics' and what might break otherwise by including the /Ldot/ldot (0x013F, 0x0140) glyphs – similarly for the f_f_i instead of uniFB03?

**Reply by koeberlin** [reply]
What @justanotherfoundry says. (Thanks, Tim!) Don’t know if I’m overcautious, but I’ve always had the feeling to be on the safe side this way (especially with these compatibility characters).

**Comment by justanotherfoundry** [comment]
> Can you please clarify what you mean by preserve ‘text semantics’ and what might break otherwise by including the /Ldot/ldot (0x013F, 0x0140) glyphs – similarly for the f_f_i instead of uniFB03?

I suppose he refers to the question whether (or which) OT features are allowed to change the semantics of the text. There is a good explanation of this issue in this tutorial: https://glyphsapp.com/learn/superscript-and-subscript-figures It’s about superscript and subscript but the reasoning should be the same.

**Comment by justanotherfoundry** [comment]
It also depends a bit on your attitude towards making fonts:

• idealistic: “These Unicode characters should not be used at all! If I include them I encourage their use. Let’s all omit them in our fonts so that users learn what’s wrong.”

• perfectionist: “I want my fonts to work as well as they can in all possible scenarios, including usage that we should consider wrong.”

• pragmatic: “Thankfully, hardly anyone is using these characters nowadays so I’ll just skip them instead of wasting my time.”
